cluster-scope/base/user.openshift.io/groups/octo-ushift-dev/group.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: user.openshift.io/v1
+kind: Group
+metadata:
+    name: octo-ushift-dev
+users:
+  - fzdarsky
+  - cooktheryan
+  - oglok
+  - screeley44
```

Comment:
CI is complaining about inconsistent indentation (`.metadata.name` vs `.users[]`. Please either indent the users by 4 spaces or name by 2.